coursedata/adventures/nl.yaml

```diff
@@ -2016,20 +2016,20 @@ adventures:
                 start_code: "print Ontsnap uit het spookhuis!"
             16:
                 story_text: |
-                    ## Haunted House Game
+                    ## Spookhuis
                     Deze spookhuisgame gebruikt de connectie tussen de verschillende lijstjes, die je in dit level kunt gebruiken.
                     Bijvoorbeeld: alle eigenschappen die bij de zombie horen staan als eerste in alle lijstjes, de heks als tweede en de vampier derde.
                     Kijk maar naar de code.
 
-                    ## Example Hedy Code
+                    ## Voorbeeld Hedy code
                     ```
                     getallen = [1, 2, 3]
                     i is getallen[random]
                     hint = ['gegrom', 'kakelende lach', 'vleermuis gefladder']
                     monsters = ['zombie', 'heks', 'vampier']
                     noodlot = ['Je brein wordt opgepeuzeld', 'Je wordt vervloekt', 'Je wordt gebeten']
                     goedlot = ['Je gooit de ham naar de zombie die er lekker aan begint te knagen.', 'Je zet de gordijnen in de fik. De heks vlucht, bang voor het vuur', 'De vampier haat knoflook en vlucht weg']
-                    wapens = ['ham', 'aansteker', 'knoflook']
+                    wapens = ['rauwe ham', 'aansteker', 'knoflook']
```

Comment:
Misschien zie ik iets over het hoofd maar ik snap niet meteen waarom hier rauwe ham moet staan? Is het omdat zombies alleen rauw eten lusten?

Reply:
In de originele code staat 'ham' als wapen en in de 'print tekst' staat een 'rauwe ham' als een van de keuzes. Als je 'rauwe ham' als antwoord geeft bij de zombie geeft hij dan aan dat je het verkeerde wapen hebt, omdat je niet ham hebt getypt. Je kunt natuurlijk ook rauwe ham in de 'print tekst' veranderen in ham, zodat deze overeenkomt met de keuzes. Dat lost het "probleem" ook op.

Snap je?

Reply:
![lvl 16 Hedy](https://user-images.githubusercontent.com/80593681/150642317-59647d59-dd21-44dd-9283-5621546b9502.jpg)

Reply:
Ah got it, helemaal duidelijk zo!